specs/Standards/NonFungibleToken/payouts.md

```diff
@@ -0,0 +1,58 @@
+# Standard for a Multiple-Recipient-Payout mechanic on NFT Contracts
+
+A standard for supporting structures that allow for NFT's to inform financial
```

Comment:
```suggestion
A standard for supporting structures that allow for NFTs to inform financial
```

Apostrophe only needed here to indicate possession. We wouldn't write "Non-Fungible Token's," so we don't write "NFT's."

Change needed thoughout document.